packages/experimental/Avatar/ios/AvatarViewManager.swift

```diff
@@ -15,12 +23,12 @@ class AvatarViewManager: RCTViewManager {
 	override func constantsToExport() -> [AnyHashable : Any]! {
 		return [
 			"sizes" : [
-				"xSmall" : AvatarSize.extraSmall.size.width,
-				"small" : AvatarSize.small.size.width,
-				"medium" : AvatarSize.medium.size.width,
-				"large" : AvatarSize.large.size.width,
-				"xLarge" : AvatarSize.extraLarge.size.width,
-				"xxLarge" : AvatarSize.extraExtraLarge.size.width
+				"xSmall" : 16,
```

Comment:
If these numbers are supposed to be in sync with something then it's probably worth a comment as to where we keep them in sync (e.g. FUA Avatar)

Reply:
That sounds good. I can at the very least leave a comment here why these are exported and where they come from.

Reply:
I added a comment. Thanks for the pushing this to be more clear.